axum-login/src/sqlx_store.rs

```diff
@@ -87,22 +85,16 @@ macro_rules! impl_user_store {
             type Error = sqlx::error::Error;
 
             async fn load_user(&self, user_id: &UserId) -> Result<Option<Self::User>, Self::Error> {
-                let mut connection = self.pool.acquire().await?;
-
                 let user: Option<User> = sqlx::query_as(&self.query)
                     .bind(&user_id)
-                    .fetch_optional(&mut connection)
```

Comment:
this appears to be one of the changes in `sqlx v0.7`, they do the acquire for you? didn't manually test but `cargo test` hopefully covers this